src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -1376,7 +1382,7 @@ protected override void GetItem(string path)
 
         private FileSystemInfo GetFileSystemItem(string path, ref bool isContainer, bool showHidden)
         {
-            path = NormalizePath(path);
+            path = GetCorrectCasedPath(path);
```

Comment:
I see we've swapped a bunch of calls to use GetCorrectCasedPath -- are there cases where NormalizePath is still called on its own? Or is it only used in the GetCorrectCasedPath method now?

Reply:
After the PR:

- GetCorrectCasedPath - 16 uses.
- NormalizePath - 8 uses.